src/components/content-tab-assistant.tsx

```diff
@@ -203,12 +209,12 @@ const AuthenticatedView = memo(
 		const RenderLastMessage = useCallback(
 			( { message, children }: { message: MessageType; children: React.ReactNode } ) => {
 				const thinkingAnimation = {
-					initial: { opacity: 0, y: 20 },
+					initial: shouldAnimate ? { opacity: 0, y: 20 } : { opacity: 1, y: 0 },
 					animate: { opacity: 1, y: 0 },
 					exit: { opacity: 0, y: -20 },
 				};
 				const messageAnimation = {
-					initial: { opacity: 0, y: 20 },
+					initial: shouldAnimate ? { opacity: 0, y: 20 } : { opacity: 1, y: 0 },
```

Comment:
Let's keep these values static and change the initial in the component prop. Line 247.

```suggestion
					initial: { opacity: 0, y: 20 } ,
```